index.js

```diff
@@ -1,51 +1,46 @@
 'use strict';
 
 const cp = require('child_process');
-const crypto = require('crypto');
 
 const CONTAINER = 'devsnek/js-eval';
 
+class Output {
+  constructor(stdout, err) {
+    this.stdout = stdout;
+    if (err) {
+      this.killed = err.killed;
+      this.code = err.code;
+      this.signal = err.signal;
+    }
+  }
+  toString() {
+    return this.stdout;
+  }
+}
+
 module.exports = (code, { environment = 'node-cjs', timeout, cpus, memory, net = 'none' } = {}) =>
   new Promise((resolve, reject) => {
-    const name = `jseval-${crypto.randomBytes(8).toString('hex')}`;
```

Comment:
i believe the name still needs to be rolled? there could be multiples of this running at once.

Reply:
yea, but deleting all of them is easy with `docker rm $(docker ps -qf label=jseval)`. It'd also work by name prefix `docker rm $(docker ps -qf name=jseval-)`